crates/holochain/src/conductor/tests/install_app_bundle.rs

```diff
@@ -238,94 +237,6 @@ async fn reject_duplicate_app_for_same_agent() {
     assert!(valid_install_of_second_app.is_ok());
 }
 
-#[tokio::test(flavor = "multi_thread")]
-async fn can_install_app_a_second_time_using_nothing_but_the_manifest_from_app_info() {
```

Comment:
The purpose of this test already got defeated with PR [4881](https://github.com/holochain/holochain/pull/4881/files#diff-49d6b93c91c1c50fddc09ed5c9db70676e39c167192bc46d1219b71fd62ceb58) by passing actual resources to the AbbBundle on line 312 (line 325 at the time of the PR).
The test intended to test that an app bundle could be provided without any resources but with the `installed_hash` field specified for the role(s) and the conductor would automatically pick an existing `DnaFile` from the ribosome store using the `installed_hash` from the app manifest as an index key.
With this PR this is now also officially not supported anymore, not just practically :)